src/System.Runtime.WindowsRuntime/src/System/IO/UnmanagedMemoryStreamInternal.cs

```diff
@@ -226,10 +222,6 @@ public override long Position
             {
                 if (!_isOpen) throw new ObjectDisposedException(null, SR.ObjectDisposed_StreamClosed);
 
-                // Note: subtracting pointers returns an Int64.  Working around
-                // to avoid hitting compiler warning CS0652 on this line.
-                if (new IntPtr(value - _mem).ToInt64() > UnmanagedMemStreamMaxLength)
-                    throw new ArgumentOutOfRangeException("offset", SR.ArgumentOutOfRange_UnmanagedMemStreamLength);
```

Comment:
A quick search shows this exception message only being used in this file. We should remove it from the SR file if that's the case.